geopandas/io/file.py

```diff
@@ -51,7 +53,6 @@ def read_file(filename, **kwargs):
     -------
     geodataframe : GeoDataFrame
     """
-    bbox = kwargs.pop('bbox', None)
```

Comment:
Given that `bbox` is not a kwarg accepted by `fiona`, I agree that moving `bbox` to the function definition makes sense.